Dockerfile

```diff
@@ -1,7 +1,48 @@
-FROM scratch
-COPY mapserver /bin/mapserver
+#== Define versions here.
+ARG ALPHINE_VER=3.20
```

Comment:
the typo aside: this won't work very well together with the renovate bot (see pull-requests)

for reference, this is how it worked before i shortened the `Dockerfile`: https://github.com/minetest-mapserver/mapserver/blob/c42a44c3c39dd414b2ba7e7d82a36f3751c08729/Dockerfile